src/AnimatedText.re

```diff
@@ -13,6 +13,7 @@ module Styles = {
 let%component make = (~delay: Time.t, ~text: string, ()) => {
   let%hook (yOffset, _state, _reset) =
     Hooks.animation(
+      ~name="yOffset",
```

Comment:
not sure if I'm using this correctly, are they just supposed to be global ids?

Reply:
They're just names to identify them in the logs I believe.